x-pack/test_serverless/api_integration/test_suites/common/console/proxy_route.ts

```diff
@@ -7,38 +7,51 @@
 
 import expect from '@kbn/expect';
 import { FtrProviderContext } from '../../../ftr_provider_context';
+import { RoleCredentials } from '../../../../shared/services';
 
 export default function ({ getService }: FtrProviderContext) {
-  const supertest = getService('supertest');
   const svlCommonApi = getService('svlCommonApi');
+  const svlUserManager = getService('svlUserManager');
+  const supertestWithoutAuth = getService('supertestWithoutAuth');
+  let roleAuthc: RoleCredentials;
 
   describe('POST /api/console/proxy', () => {
+    before(async () => {
+      roleAuthc = await svlUserManager.createApiKeyForDefaultRole();
+    });
+    after(async () => {
+      await svlUserManager.invalidateApiKeyForRole(roleAuthc);
+    });
     describe('system indices behavior', () => {
       it('returns warning header when making requests to .kibana index', async () => {
-        return await supertest
+        return await supertestWithoutAuth
           .post('/api/console/proxy?method=GET&path=/.kibana/_settings')
           .set('kbn-xsrf', 'true')
           .set(svlCommonApi.getInternalRequestHeader())
+          .set(roleAuthc.apiKeyHeader)
           .then((response) => {
             expect(response.header).to.have.property('warning');
-            const { warning } = response.header as { warning: string };
-            expect(warning.startsWith('299')).to.be(true);
-            expect(warning.includes('system indices')).to.be(true);
+            // TODO: response.header.warning is an empty string: `''` with my changes.
```

Comment:
@yuliacech Should this it block be dropped or?

Reply:
hey @wayneseymour, it looks like the api keys have somewhat different permissions as the test used before and that causes no system indices to be returned in the response. if that is intended I think it's okay to delete that assumption.

Reply:
cool beans.  thanks very much Yulia!

Reply:
About to push the update, but it makes me think this test should be change more, or deleted.  
What say you @yuliacech ?

Reply:
@wayneseymour I think if there is no warning about system indices, it's okay to delete the `it` block altogether.